msg.go

```diff
@@ -528,10 +556,12 @@ func unpackTxt(msg []byte, off0 int) (ss []string, off int, err error) {
 func isDigit(b byte) bool { return b >= '0' && b <= '9' }
 
 func dddToByte(s []byte) byte {
+	_ = s[2]
```

Comment:
what does this accomplish?

Reply:
It's all about bounds check elimination. I'll add a comment to clarify. Each of `s[0]`, `s[1]` and `s[2]` will generate a bounds check, but this line allows the compiler to prove the other two can't fail and eliminate those checks.

Without it each of these functions has three separate bounds checks:
```
./msg.go:560:16: Found IsInBounds
./msg.go:560:33: Found IsInBounds
./msg.go:560:49: Found IsInBounds
./msg.go:565:16: Found IsInBounds
./msg.go:565:33: Found IsInBounds
./msg.go:565:49: Found IsInBounds
```

With it they have just one each:
```
./msg.go:559:7: Found IsInBounds
./msg.go:564:7: Found IsInBounds
```